File: eCarService.Model/CarService.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace eCarService.Model
{
    public class CarService
    {
        public int CarServiceId { get; set; }
        public string Name { get; set; }
        public string Address { get; set; }
        public DateTime DateCreated { get; set; }
        public string PhoneNumber { get; set; }
        public int UserId { get; set; }


    }
}

File: ecarservice_mobile/lib/model/carService.dart
import 'package:json_annotation/json_annotation.dart';
part 'carService.g.dart';

@JsonSerializable()
class CarService {
  int? carServiceId;
  String? name;
  String? address;
  DateTime? dateCreated;
  String? phoneNumber;
  int? userId;

  CarService() {}

  factory CarService.fromJson(Map<String, dynamic> json) =>
      _$CarServiceFromJson(json);

  /// Connect the generated [_$CarServiceToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$CarServiceToJson(this);
}
